aggregator/src/aggregator.rs

```diff
@@ -372,9 +373,16 @@ impl<C: Clock> Aggregator<C> {
         Ok((encoded_hpke_config_list, signature))
     }
 
-    async fn handle_upload(&self, task_id: &TaskId, report_bytes: &[u8]) -> Result<(), Arc<Error>> {
-        let report =
-            Report::get_decoded(report_bytes).map_err(|err| Arc::new(Error::MessageDecode(err)))?;
+    async fn handle_upload(
+        &self,
+        task_id: &TaskId,
+        report_bytes: &[u8],
```

Comment:
We should change this argument's name as well.
```suggestion
        request_bytes: &[u8],
```